lib/hap/encrypted_http_server.ex

```diff
@@ -82,6 +82,13 @@ defmodule HAP.EncryptedHTTPServer do
     end
   end
 
+  # Note that we do not enforce timed writes per 6.7.2.4; we just throw this request on the ground
+  put "/prepare" do
+    conn
+    |> put_resp_header("content-type", "application/hap+json")
+    |> send_resp(200, Jason.encode!(%{status: 0}))
```

Comment:
hah! thats what that need to be... I had tried something very similar but not exactly the same... 🤦 


I'll give this a shot!